developers/issuers.md

```diff
@@ -14,7 +14,7 @@ CHAPI integrates easily into issuer websites, allowing your site to issue Verifi
 If you're working in vanilla JavaScript, you can add the `navigator.credentials` and `credentialHandlerPolyfill` globals to your code and then load the polyfill library:
 
 ```html
-<script src="https://unpkg.com/credential-handler-polyfill@3.0.1/dist/credential-handler-polyfill.min.js"></script>
+<script src="https://unpkg.com/credential-handler-polyfill/dist/credential-handler-polyfill.min.js"></script>
```

Comment:
Hmm, well, when using a script tag, people might want to lock to at least a major version. We'll want to make note of that somewhere.

Reply:
Yeah. Agreed. I'll push up a tweak to pin them at `3` for now.